src/containers/treePage/index.tsx

```diff
@@ -81,7 +81,6 @@ const TreeCareContainer = styled.div`
 const MobileTreeCareContainer = styled.div`
   margin-top: 5vh;
   border: solid 1px ${LIGHT_GREY};
-  max-height: 50vh;
```

Comment:
This is to address an issue on vertical mobile screens - screen still looks a little rough around the edges, maybe we should touch up mobile stylings when we get the chance?
Before:

![1](https://user-images.githubusercontent.com/68914997/215004950-5ebec7ea-7d30-4230-9a8c-931b25f49110.png)

After:

![2](https://user-images.githubusercontent.com/68914997/215005301-5d1eaf1c-0d2b-4deb-911b-54dbe528b120.png)

Reply:
💯 Nice! And yeah, we should def take a look at this in the future